### DISCUSSION TITLE
Use a different font through @carbon/styles

### DISCUSSION THREAD

**Discussion by lSilvani** [carbon-design-system/carbon#13910] [Help] [answered]
Hello there, I am trying to understand how I can use a font other than IBM Plex through @carbon/styles, but I am struggling a bit. Both the styles and type docs talk about how to use the available styles in my own selectors but I cannot find info on how to use a different font. Can anyone help out on this? Thank you! Using styles v 1.30

**Comment by tw15egan** [comment, marked as answer]
Helped over in https://github.com/carbon-design-system/carbon/issues/13906

Example can be found here: https://stackblitz.com/edit/github-hf5kjr-sfmrgv?file=src%2Findex.scss,src%2F_types.scss
